UE/Application/Ports/BtsPort.cpp

```diff
@@ -37,7 +37,7 @@ void BtsPort::handleMessage(BinaryMessage msg)
     {
         common::IncomingMessage reader{msg};
         auto msgId = reader.readMessageId();
-        auto from = reader.readPhoneNumber();
+        common::PhoneNumber from = reader.readPhoneNumber();
```

Comment:
Po co ta zmiana?

Reply:
Bo error miałem taki, że readPhoneNumber zwracało integer. Nie wiem o co chodziło, ale jak zmieniłem na common:PhoneNumber to zaczęło działać.